BLSQ/openhexa-frontend
# Feat pipeline version configration nf

## Changes
Adds pipeline version configuration dialog ( configuration that contains parameter values) . 
Adds button to edit pipeline version configuration on pipeline version page, and pipeline versions list page.
Displays configuration values in pipeline version on a pipeline page and pipeline versions page. 

## How/what to test
- Go to workspaces/<workspace_name>/pipelines/<pipeline_name>/  :  you will see the configuration of pipeline parameters in the table displaying pipeline parameters. You can edit the configuration by clicking on the button "Edit config". 
- Go to workspaces/<workspace_name>/pipelines/<pipeline_name>/versions/ : you will the the configuration of pipeline parameters in the table displaying pipeline parameters for each version of the pipeline by clicking on "Edit config"
## Screenshots / screencast
<img width="1097" alt="Screenshot 2024-05-31 at 14 28 08" src="https://github.com/BLSQ/openhexa-frontend/assets/4411921/26210bed-e297-4558-bfee-c027254c325f">
<img width="1106" alt="Screenshot 2024-05-31 at 14 28 15" src="https://github.com/BLSQ/openhexa-frontend/assets/4411921/b5c2fb3b-439c-481e-99db-67f3237b43b0">
<img width="1101" alt="Screenshot 2024-05-31 at 14 28 22" src="https://github.com/BLSQ/openhexa-frontend/assets/4411921/8cd75b9b-d848-4161-9d38-9a99accf941e">
<img width="1099" alt="Screenshot 2024-05-31 at 14 28 27" src="https://github.com/BLSQ/openhexa-frontend/assets/4411921/f781c285-be1f-42d8-a0e5-8dc8d870af8b">
<img width="1109" alt="Screenshot 2024-05-31 at 14 28 38" src="https://github.com/BLSQ/openhexa-frontend/assets/4411921/eae65722-8621-46d1-886d-e4626c94280a">
